thinc/loss.py

```diff
@@ -47,12 +49,15 @@ def __init__(
         missing_value: Optional[Union[str, int]] = None,
         neg_prefix: Optional[str] = None,
         label_smoothing: float = 0.0,
+        class_weights: Optional[List[float]] = None,
```

Comment:
It would be nice if it was also possible to pass the class weights as `Floats1d`.